ui/app/views/deployments/new.html

```diff
@@ -215,6 +218,39 @@ <h1 class="page-header">New Deployment</h1>
             </div>
         </div>
 
+        <div class="col-lg-12 col-sm-12 animate-switch-container" ng-switch-when="choose-commit">
+            <div class="panel panel-default">
+                <div class="panel-body">
+                    <h2>Enter your commit sha</h2>
+                    <input class="form-control" id="commitSha" type="text" placeholder="Commit sha" ng-model="commitSha" ng-change="getSuitableDeployableVersionsFromPartialSha(commitSha)" autofocus required>
+
+                    <div class="table-responsive">
+                        <table class="table table-hover" datatable="ng" dt-options="dtOptionsDeployableVersion" dt-column-defs="dtColumnDefsDeployableVersion">
+                            <thead>
+                            <tr>
+                                <th>Date</th>
+                                <th>Commit</th>
+                                <th>Message</th>
+                                <th>Author</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            <tr ng-repeat="deployableVersion in myDeployableVersions"
```

Comment:
As discussed verbally, lets not duplicate everything and just make this feature work like the "deploy latest from"